src/Lecture1.hs

```diff
@@ -67,7 +71,10 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n
+    | n < 0 = mod (n * (-1)) 10
+    | n >= 0 = mod n 10
```

Comment:
That's correct! (although, with a GHC warning)
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```